README.rst

```diff
@@ -56,7 +56,69 @@ Features
 Installation
 ============
 
-TBD (Helm)
+Requirements
+------------
+
+In order to install ETOS you need to meet the following requirements.
+
+- An up and running kubernets cluster (`<https://kubernetes.io/>`_)
+- Helm version 3.x installed (`<https://helm.sh/>`_)
+
+
+Installation Steps
+------------------
+
+1. First we need to add helm repository where the etos helm charts are stored
```

Comment:
Consider:
First, we need to add a Helm repository where the ETOS Helm charts are stored

Reply:
Fixed.